clients/cobol-lsp-vscode-extension/CHANGELOG.md

```diff
@@ -1,6 +1,31 @@
 # Changelog
 All notable changes to the COBOL Language Support extension are documented in this file.
 
+## [0.12.0](https://github.com/eclipse/che-che4z-lsp-for-cobol/compare/0.11.1...0.12.0) (2020-07-07)
+
+#### Added
+
+- Support for local copybooks
+- Dynamic registration for watchers
+
+#### Changed
+
+- Adjust the info message about missing copybooks
+- Reduce the number of error/informational messages
+- Inform user about wrong credentials and stop sending other MF requests
+- Copybooks downloaded from the mainframe are now downloaded to /.c4z/.copybooks in the workspace
+- Settings parameter for DSN paths changed
+- Updates to README
+
+#### Fixed
+
+- Create a .gitignore file if it doesn't exist
+- Fix glob pattern for dynamic registration
+- Fix issue with indirect overwrite settings conf for DSN paths
+- Fix issues related to unrecognised copybook extensions
+- Fix representation of settings.json in Theia
+- Use sync APIs to create a folder and write to a file to avoid unexpected behavior
```

Comment:
This is applicable only for test and doesn't affect the user experience, so we may drop it

Reply:
done in latest commit